build/remote-config/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+  // Note: settings in this file will override user-level settings in the workspace.
+  //       Any setting that may be changed by the user should not be set here.
+  "git.defaultCloneDirectory": "/projects/"
```

Comment:
@azatsarynnyy @amisevsk 
an alternative way:
put `projects` [here](https://github.com/che-incubator/che-code/blob/6684f6d4ff883edbc05bf6f9950dafa293649cb9/code/extensions/git/package.json#L2348)

this way it's possible for a user to change the value for another one in settings and it will not be overwritten after restarting a workspace

I don't insist, just FYI

Reply:
Thanks @RomanNikitenko for mentioning another approach.
Yes, this is how we usually do it, e.g.: https://github.com/che-incubator/che-code/pull/245/files
IMO, a couple of the pros of the approach @amisevsk provided in this PR, is that it allows keeping the VS Code sources untouched and storing all overridden default settings in a single file.

Reply:
I was generally trying to avoid changing anything in a subdir we don't own (I don't want to break the rebase job or anything like that) but it seems like this would be a relatively safe change.

I'm happy to do either here, whatever makes more sense :+1: